core-aam/core-aam.html

```diff
@@ -1208,7 +1208,12 @@
 				<tr id="role-map-scrollbar">
 					<th><a class="role-reference" href="#scrollbar"><code>scrollbar</code></a></th>
           <td><code>ROLE_SYSTEM_SCROLLBAR</code></td>
-          <td><code>scrollbar</code></td>
+          <td>
+            <ul>
+              <li>Control Type = <code>ScrollBar</code></li>
```

Comment:
Pardon my delay. No objections on the style I would gladly follow the established guidelines. I did the change as I've noticed many UIA items had inconsistent style already and wanted to pick some consistent style. Should have asked!

I'll resolve conflict soon, hopefully tomorrow...